webrtc.html

```diff
@@ -4745,7 +4745,7 @@ <h2>Methods</h2>
             <dt><dfn data-idl><code>generateCertificate</code></dfn>, static</dt>
             <dd>
               <p>The <code>generateCertificate</code> function causes the
-              <a>user agent</a> to create and store an X.509 certificate
+              <a>user agent</a> to create an X.509 certificate
```

Comment:
The generateCertificate function doesn't store a certificate (except into an internal slot). It creates a certificate that can be stored.